### ISSUE TITLE
Metadata for equations of state

### ISSUE THREAD

**Issue by kemattil** [simphony/simphony-metadata#80]
The following metadata support multiphase modelling of fluid flows:

  EQUATION_OF_STATE:
    definition: Equation of state for modelling multiphase fluids
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]

  IDEAL_GAS:
    parent: CUBA.EQUATION_OF_STATE
    definition: Equation of state for and ideal gas
    models: [CUBA.CONTINUUM]

  VAN_DER_WAALS:
    definition: Van der Waals equation of state
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]
    CUBA.MOLAR_MASS:
      default: 1.0
    CUBA.CRITICAL_PRESSURE:
      default: 1.0
    CUBA.CRITICAL_TEMPERATURE:
      default: 1.0
    CUBA.CRITICAL_DENSITY:
      default: 1.0
    CUBA.CRITICAL_MOLAR_VOLUME:
      default: 1.0
    CUBA.CRITICAL_SPECIFIC_VOLUME:
      default: 1.0

  REDLICH_KWONG:
    definition: Redlich-Kwong equation of state
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]
    CUBA.MOLAR_MASS:
      default: 1.0
    CUBA.CRITICAL_PRESSURE:
      default: 1.0
    CUBA.CRITICAL_TEMPERATURE:
      default: 1.0
    CUBA.CRITICAL_DENSITY:
      default: 1.0
    CUBA.CRITICAL_MOLAR_VOLUME:
      default: 1.0
    CUBA.CRITICAL_SPECIFIC_VOLUME:
      default: 1.0

  SOAVE:
    definition: Soave equation of state
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]
    CUBA.MOLAR_MASS:
      default: 1.0
    CUBA.CRITICAL_PRESSURE:
      default: 1.0
    CUBA.CRITICAL_TEMPERATURE:
      default: 1.0
    CUBA.CRITICAL_DENSITY:
      default: 1.0
    CUBA.CRITICAL_MOLAR_VOLUME:
      default: 1.0
    CUBA.CRITICAL_SPECIFIC_VOLUME:
      default: 1.0
    CUBA.ACENTRIC_FACTOR:
      default: 0.344

  PENG_ROBINSON:
    definition: Peng-Robinson equation of state
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]
    CUBA.MOLAR_MASS:
      default: 1.0
    CUBA.CRITICAL_PRESSURE:
      default: 1.0
    CUBA.CRITICAL_TEMPERATURE:
      default: 1.0
    CUBA.CRITICAL_DENSITY:
      default: 1.0
    CUBA.CRITICAL_MOLAR_VOLUME:
      default: 1.0
    CUBA.CRITICAL_SPECIFIC_VOLUME:
      default: 1.0
    CUBA.ACENTRIC_FACTOR:
      default: 0.344

  CARNAHAN_STARLING:
    definition: Carnahan-Starling equation of state
    parent: CUBA.MATERIAL_RELATION
    models: [CUBA.CONTINUUM]
    CUBA.MOLAR_MASS:
      default: 1.0
    CUBA.CRITICAL_PRESSURE:
      default: 1.0
    CUBA.CRITICAL_TEMPERATURE:
      default: 1.0
    CUBA.CRITICAL_DENSITY:
      default: 1.0
    CUBA.CRITICAL_MOLAR_VOLUME:
      default: 1.0
    CUBA.CRITICAL_SPECIFIC_VOLUME:
      default: 1.0

**Comment by ahashibon** [comment]
Thanks @kemattil, looks good but can the prefix "CRITICAL" be avoided? if so we can reuse CUBA.TEMPERATURE etc.

**Comment by kemattil** [comment]
No, e.g. REDUCED_TEMPERATURE = (ACTUAL_)TEMPERATURE/CRITICAL_TEMPERATURE. That is,
there are three distinct temperature variables. The same for other variables ...

**Comment by ahashibon** [comment]
VAN_DER_WAALS, REDLICH_KWONG,  SOAVE, PENG_ROBINSON, CARNAHAN_STARLING, --> parent should be CUBA.EQUATION_OF_STATE

**Comment by kemattil** [comment]
Yes, you are right. My mistake ...
